decidim-core/app/views/decidim/user_activities/index.html.erb

```diff
@@ -7,3 +7,11 @@
   resource_types: resource_types,
   user: user
 } %>
+
+<%=
+  render partial: "decidim/shared/flag_user_modal", locals: {
+      reportable: user,
+      form: report_form,
+      url: decidim.report_user_path(sgid: user.to_sgid.to_s)
+  }
```

Comment:
Shouldn't this be rendered by `decidim-core/app/cells/decidim/author/flag_user.erb`?

Reply:
The tricky part is that report_form, that is being initialized in the controller part.